build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
This file should be name `build-spike-proto.sh` or something more specific than just `build.sh`.

Reply:
done.